### DISCUSSION TITLE
React Native Facebook SDK "Needs Update" Workaround

### DISCUSSION THREAD

**Discussion by carlreiser2** [thebergamo/react-native-fbsdk-next#481] [Q&A]
Hello,

I am trying to integrate this package so that I can track my advertisement events in the facebook business console, specifically App Installs; however, while I am seeing Events in the console for ActivateApp tagged with Integration Facebook SDK integration, I am not seeing App Installs. The App Install is tagged with MMP (relating to my AppsFlyer integration). I also see in the console that I have the issue that the package needs to be updated. I've seen some people answer online that v11.3.0 fixes the issue and passes facebooks filters. I have tried to downgrade the package but still see in the dashboard that I need to update the package. I was wondering if anyone has any more information on how to get around the issue via downgrading or some other method. What should I do to track App Installs via the Facebook SDK Integration

Thanks!

**Comment by mikehardy** [comment]
#476 or #452 perhaps? Did you search the issues list?
